### ISSUE TITLE
Can't delete virtual machine due to JavaScript error

### ISSUE THREAD

**Issue by Vogelhaus** [cockpit-project/cockpit-machines#181]
I am on Cockpit Version 244.1.
The module cockpit-machines is of the following version:
```
Name         : cockpit-machines
Version      : 243.1
Release      : 1.fc34
Architecture : noarch
Size         : 6.7 M
Quelle       : cockpit-machines-243.1-1.fc34.src.rpm
Repository   : @System
Aus Paketque : updates
Summary      : Cockpit user interface for virtual machines
URL          : https://github.com/cockpit-project/cockpit-machines
Lizenz       : LGPLv2+ and MIT
Description  : Cockpit component for managing virtual machines.
             : 
             : If "virt-install" is installed, you can also create new virtual
             : machines.

```
I found a problem that I can't delete created virtual machines.

Steps to reproduce:
1. Go to Virtual Machines Menu
2. Select virtual machine you want to remove
3. Click on "Burger Menu" and click "Delete"
4. Modal dialog appears that asks for deletion of associated storage volume, check the checkbox
5. Press on red "Delete" button
6. "Oops" Message appears in the top header 

Javascript console ouput
`[Error] TypeError: v.a.all is not a function. (In 'v.a.all(e)', 'v.a.all' is undefined)
	st (index.js:2:759878)
	pt (index.js:2:761028)
	Y (index.js:2:857812)
	B (index.js:2:745449)
	Jt (index.js:2:766985)
	Qt (index.js:2:766203)
	Qt
	(anonyme Funktion) (index.js:2:863849)
	R (index.js:2:857590)
	Kt (index.js:2:766018)
	Kt`

**Comment by KKoukiou** [comment]
Hi @Vogelhaus see https://github.com/cockpit-project/cockpit-machines/issues/178
This is fixed in version 244.1, which you can get for [Fedora 34](https://bodhi.fedoraproject.org/updates/FEDORA-2021-380660d6cd) or [Fedora 33](https://bodhi.fedoraproject.org/updates/FEDORA-2021-15d8576d18) in updates-testing.

Sorry for this.

**Comment by Vogelhaus** [comment]
@KKoukiou 
Thank you for your prompt reply. I updated as you advised and it works now! Thank you, I think we can close this issue.
